src/pages/settings/Security/TwoFactorAuth/TwoFactorAuthPage.js

```diff
@@ -0,0 +1,81 @@
+import React, {useCallback, useEffect, useState} from 'react';
+import {withOnyx} from "react-native-onyx";
+import PropTypes from "prop-types";
+import CodesStep from "./Steps/CodesStep";
+import DisableStep from "./Steps/DisableStep";
+import IsEnabledStep from "./Steps/IsEnabledStep";
+import VerifyStep from "./Steps/VerifyStep";
+import SuccessStep from "./Steps/SuccessStep";
+import ONYXKEYS from "../../../../ONYXKEYS";
+import CONST from "../../../../CONST";
+import * as TwoFactorAuthActions from '../../../../libs/actions/TwoFactorAuthActions';
+import TwoFactorAuthContext from "./TwoFactorAuthContext";
+
+const propTypes = {
+    account: PropTypes.shape({
+        /** Whether or not the user has two factor authentication enabled */
+        requiresTwoFactorAuth: PropTypes.bool,
+
+        /** The current step in the two factor authentication process */
+        twoFactorAuthStep: PropTypes.string,
+    }),
+}
+
+const defaultProps = {
```

Comment:
Use prop destructuring with default values instead of `defaultProps` -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring